src/api/app/views/webui/attribute/_attributes.html.haml

```diff
@@ -14,7 +14,10 @@
           %td{ class: "#{attribute.namespace}-#{attribute.name}" }
             = attribute.fullname
           %td{ class: "#{attribute.namespace}-#{attribute.name}-values" }
-            = attribute.values.map(&:value).join(', ')
+            %ul
+            - attribute.values.each do |v|
+              %li
+                %pre{ style: "display: inline-flex"}= v.value
```

Comment:
what about a class for this instead of inline style?

Reply:
you can redo all this in webui2 - for now I made this low effort